helpers/README.md

```diff
@@ -0,0 +1,353 @@
+
+---
+## Matrix Macros
+```c
+#define Matrix_RotateY_s(binang, A) Matrix_RotateY(BINANG_TO_RAD(binang), A)
+#define Matrix_RotateX_s(binang, A) Matrix_RotateX(BINANG_TO_RAD(binang), A)
+#define Matrix_RotateZ_s(binang, A) Matrix_RotateZ(BINANG_TO_RAD(binang), A)
+
+#define Matrix_RotateY_f(degf, A)   Matrix_RotateY(DEGF_TO_RADF(degf), A)
+#define Matrix_RotateX_f(degf, A)   Matrix_RotateX(DEGF_TO_RADF(degf), A)
+#define Matrix_RotateZ_f(degf, A)   Matrix_RotateZ(DEGF_TO_RADF(degf), A)
+```
+[Back to top](#list-of-content)
+
+---
+# Physics
+---
+## PhysicsStrand Struct
+```md
+typedef struct {
+	PhysicsInfo       info;
+	PhysicsHead       head;
+	PhysicsGfx        gfx;
+	PhysicsSpheres    spheres; // "collision" spheres
+	PhysicsConstraint constraint;
+	PhysicsRigidity   rigidity;
+	f32 limbsLength[];
+} PhysicsStrandInit;
+
+typedef struct {
+	PhysicsInfo       info;
+	PhysicsHead       head;
+	PhysicsGfx        gfx;
+	PhysicsSpheres    spheres; // "collision" spheres
+	PhysicsConstraint constraint;
+	PhysicsRigidity   rigidity;
+	f32* limbsLength;
+} PhysicsStrand;
+```
+Difference between `PhysicsStrand` and `PhysicsStrandInit` is like how `Zelda 64` games handle `Colliders`.
+
+[Back to top](#list-of-content)
+
+---
+### PhysicsInfo
+```c
+typedef struct {
+	s32 numLimbs;
+	f32 gravity;
+	f32 floorY;  // world.pos.y, won't go through this
+	f32 maxVel;  // Clamps velocity value
+	f32 velStep; // Values below 1.0f will give it spring like motion
+	f32 velMult; // Control the power of velocity
+} PhysicsInfo;
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsHead
+```c
+typedef struct {
+	Vec3f pos;
+	Vec3s rot;
+	MtxF* mtxF;
+} PhysicsHead;
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsGfx
+```c
+typedef struct {
+	u32   dlist;
+	Vec3f scale;  // Gfx scale
+	u8    segID;  // For matrix
+	u8    noDraw; /* If there needs to be calculations
+	                 to get in position before drawing */
+} PhysicsGfx;
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsSpheres
+```c
+typedef struct {
+	s32    num; // amount of spheres
+	Vec3f* centers;
+	f32    radius;
+} PhysicsSpheres;
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsConstraint
+```c
+typedef struct {
+	u8    lockRoot;    // Prevent physics rotating root limb
+	Vec2f rotStepCalc; // DEG, limits rot to next limb in main calc
+	Vec2f rotStepDraw; // DEG, limits rot on draw, smoothens output
+} PhysicsConstraint;
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsRigidity
+```c
+typedef struct {
+	s32   num;  // How many limbs will be smoothed with push
+	Vec3f push; // direction Z, pushes based on rot[0]
+	f32   mult; // How much the pushing fill affect
+	Vec3f rot;  // DEG, rigids towards, relative rot
+} PhysicsRigidity;
+```
+[Back to top](#list-of-content)
+
+---
+## Examples
+Here you set the arrays for physics to store/read values from. In this specific example the substruct is named `PonytailPhysics`, but you can name it whatever you like.
```

Comment:
```suggestion
Here you set the arrays for physics to store/read values from. In this specific example the substruct is named `PonytailPhysics`, but you can name it whatever you like.
```
`PonytailPhysics` ?

Reply:
There might be some oversights in readme. I'll do another check at it today!